MACM/MACM.py

```diff
@@ -438,7 +438,8 @@ def run(self):
         all_events["informationIDs"]=all_events.iloc[:,6].apply(lambda x: [ self.imapping.columns[int(id)] if id >=0 else "-1.0" for id in x])
 
         identifier = str(dt.datetime.now())
-        file_name = os.path.join(self.OUTPUT_FOLDER_PATH,"MACM_MMD{0}_Alpha{1}_{2}.csv".format(self.MAX_MEMORY_DEPTH,self.MEMORY_DEPTH_FACTOR,identifier))
+        model_type_string = 'Q' + ('P' if self.ENABLE_MODEL_P else 'X') + ('I' if self.ENABLE_MODEL_I else 'X') + ('-CA' if self.ENABLE_CONTENT_MUTATION else '-XX')
```

Comment:
@deamonpog can you use `f""` or `"".format()` here as well, please? Ideally, we should change all strings to use one of these. Thanks.

Reply:
Sure thing.